pkg/cloudprovider/fake/cloudprovider.go

```diff
@@ -53,6 +54,9 @@ type CloudProvider struct {
 
 	CreatedNodeClaims map[string]*v1beta1.NodeClaim
 	Drifted           cloudprovider.DriftReason
+	NodeClassGroup    string
+	NodeClassVersion  string
+	NodeClassKind     string
```

Comment:
Is there a reason we're making this three variables rather than one variable of type schema.GroupVersionKind?